### PR TITLE
Background audio, warnings, disable device owner

### PR DESCRIPTION
**Pull request by threethan**
**TL;DR**
- Background audio fixed & improved
- Better UI w/ More warnings
- Ability to disable device owner & uninstall Oculess
*** 
## Full Changes
**I've made a number of updates to my background audio function**
- Uses a different AppOpsHelper class which supports android 12
- Runs automatically on system startup, and should stay running more reliably. *(The button now tells users this instead of activating the feature)*
- Added a permission so that non-system apps will be scanned in android >10
- System apps are excluded
- Audio is fixed every second (previously 2 seconds)
- Apps are rescanned every minute (previously once on button press)

**The UI has been tweaked to be more clean and better warn users**
- Colors have been changed to better reflect system UI, and to improve visibility of buttons in popups
- The option to remove accounts has been properly marked as unsafe
 - The popup now explains that removing a Meta account is permanent
- The main activity now displays a text directing the user to go to GitHub for info, rather than the current status of the companion server
  - The status of the companion server is still displayed on the button and has been added to the popup it activates
- Added "Enable Device Ownership Button" that displays the following: `This can be undone, but REMOVING ACCOUNTS IS PERMANENT.\n1. Remove all accounts with the button above\n2. Using ADB: adb shell dpm set-device-owner com.bos.oculess/.DevAdminReceiver\n`
- Added "Disable Device Ownership Button" (displays in place of the enable button when applicable) that allows device owner to be deactivated, after which the app can be uninstalled normally. *This doesn't return the Meta account though*
- The option to disable updates has been removed entirely
  - Instead, I added ADB commands for this to the readme, as users would need ADB to unf*** themselves anyways
  - The button and code are still there, but it's set to `visibility:gone` in the layout so it cannot be accessed by users

**Additional Fixes:**
- App icon now displays correctly in all contexts
- The Remove Accounts button now opens the proper intent, instead of just the main page of settings
- I added snippets to the ReadMe to disable telemetry apps with ADB as an alternative to Oculess
- The ReadMe now mentions that background audio is natively supported on Quest 2 & Pro

image

### CODE REVIEW

**Comment by basti564** [comment]
Thanks for the changes. Looks good!
